rental.rb

```diff
@@ -0,0 +1,22 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- Please, here person `person` and `book` are both **getter** and **setter**. Therefore move them to `attr_accessor` instead of `attr_reader`
- Open this link to [see an example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)

Reply:
:heavy_check_mark:  person and book add to attr_accessor